GVFS/GVFS.FunctionalTests/Tests/MultiEnlistmentTests/ConfigVerbTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using GVFS.FunctionalTests.Tools;
+using GVFS.Tests.Should;
+using NUnit.Framework;
+using System.IO;
+
+namespace GVFS.FunctionalTests.Tests.MultiEnlistmentTests
+{
+    [TestFixture]
+    [NonParallelizable]
+    [Category(Categories.FullSuiteOnly)]
+    [Category(Categories.WindowsOnly)]
+    public class ConfigVerbTests : TestsWithMultiEnlistment
+    {
+        private const string ConfigFilePath = @"C:\ProgramData\GVFS\gvfs.config";
+
+        [OneTimeSetUp]
+        public void DeleteAllSettings()
+        {
+            if (File.Exists(ConfigFilePath))
+            {
+                File.Delete(ConfigFilePath);
+            }
+        }
+
+        [TestCase, Order(1)]
+        public void CreateValues()
+        {
+            this.RunConfigCommandAndCheckOutput("integerString 213", null);
+            this.RunConfigCommandAndCheckOutput("integerString", new string[] { "213" });

```

Comment:
Minor style comment, we can use implicitly typed arrays here and drop the `string`.  In other words, we can change `new string[] { "213" }` to `new[] { "213" }`